Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3619 @@
+{
```

Comment:
<p>explain what a "crop model" is after you quote it in the first sentence and then move on to the web app info</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>